server/ws/plugin_adapter.go

```diff
@@ -285,6 +295,27 @@ func (pa *PluginAdapter) getUserIDsForWorkspace(workspaceID string) []string {
 	return userIDs
 }
 
+func (pa *PluginAdapter) getUserIDsForAllWorkspaces() []string {
+	userMap := map[string]bool{}
+
+	for _, listener := range pa.listeners {
+		userMap[listener.userID] = true
+	}
+
+	userIDs := []string{}
+	for userID := range userMap {
+		userIDs = append(userIDs, userID)
+	}
+	return userIDs
+}
+
+func (pa *PluginAdapter) BroadcastConfigChange(pluginConfig model.ClientConfig) {
```

Comment:
Now that the cluster propagation PR has been merged, I think we should create two functions mimicking the `sendWorkspaceMessage` and `sendWorkspaceMessageSkipCluster` that were introduced with it like `sendMessageToAll` and `sendMessageToAllSkipCluster` and have the former called from `BroadcastConfigChange`.

I'm thinking something like:

```go
func (pa *PluginAdapter) sendMessageToAllSkipCluster(payload map[string]interface{}) {
	// This could fetch the users and loop over them if it was necessary
	pa.api.PublishWebSocketEvent(websocketActionUpdateConfig, payload, &mmModel.WebsocketBroadcast{})
}

func (pa *PluginAdapter) sendMessageToAll(payload map[string]interface{}) {
	go func() {
		clusterMessage := &ClusterMessage{Payload: payload}

		pa.sendMessageToCluster("websocket_message", clusterMessage)
	}()

	pa.sendMessageToAllSkipCluster(payload)
}

func (pa *PluginAdapter) BroadcastConfigChange(pluginConfig model.ClientConfig) {
	pa.sendMessageToAll(utils.StructToMap(pluginConfig))
}
```